EpsilonAuxBMS/Src/main.c

```diff
@@ -507,13 +504,19 @@ void HAL_CAN_RxCpltCallback(CAN_HandleTypeDef* hcan)
         // Voltages are 2 bytes each, and memory is stored in little endian format
         orionStatus.minCellVoltage = (uint16_t)msg->Data[0] | msg->Data[1] << 8; // Min Cell voltage
         orionStatus.maxCellVoltage = (uint16_t)msg->Data[3] | msg->Data[4] << 8; // Max Cell Voltage
+
         orionStatus.canMsgReceived = 1;
     }
-    else if (msg->StdId == DRIVERS_INPUTS_STDID && msg->DLC == 4)
+    else if (msg->StdId == ORION_TEMP_INFO_STDID && msg->DLC == 8)
+    {
+        orionStatus.highTemperature = msg->Data[0];
```

Comment:
I think there should be a `orionStatus.canMsgReceived = 1` in this else if block